MidiFile/MidiEvents/MidiEvent.h

```diff
@@ -20,14 +20,16 @@ class MidiEvent
 
 public:
 	MidiEvent(MIDI_EVENT event_type, TICKS delta_time, TICKS song_time)
-	: m_event_type(event_type)
+    : m_message_length(0)

```

Comment:
Members are initialized in order of declaration, and we got a warning here because the constructor made it look like the order would be different.